### PR TITLE
crm_departement : a section id change

### PR DESCRIPTION
**Pull request by vrenaville**
- on_change_section python code ported from 6.1 + add the saleman in return value, to set the salesman corresponding to the salesteam reponsible.
- remove user_id python code, as it's not call from XML, and selecting a departement base on user_id is not usefull because a user can be on multiple department.
- pep 8 cleaning of the module

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1: (code review)

File: crm_department/__init__.py [hunk 1]
```diff
@@ -20,5 +20,3 @@
 #
 ##############################################################################
 import crm
```
Comment (dreispt) [inline crm_department/__init__.py:22]:
```
from . import crm
```

**Comment by dreispt** [comment]
One small nitpick to check on, but otherwise LGTM :+1:

**Comment by lepistone** [comment]
Thanks @vrenaville, :+1:. I'm merging as soon as you make the small change suggested by @dreispt.

**Comment by lepistone** [comment]
I applied what @dreispt suggested while merging. Thanks

**Pull request closed without merging**